darts/anomaly_detection/anomaly_model.py

```diff
@@ -0,0 +1,214 @@
+from cProfile import label
```

Comment:
Don't import anything that is not used

Reply:
Pay attention to the order of the imports - see [isort](https://pycqa.github.io/isort/) and more in general [formatting and linting for Darts](https://github.com/unit8co/darts/blob/master/CONTRIBUTING.md#code-formatting-and-linting)